### PR TITLE
[CDAP-17025][CDAP-17026] Integrate New LogViewer with Pipeline Details and Pipeline Preview

### PR DESCRIPTION
**Pull request by elfenheart**
JIRAs:
- https://issues.cask.co/browse/CDAP-17025
- https://issues.cask.co/browse/CDAP-17026

Build: ~https://builds.cask.co/browse/CDAP-UDUT701~
https://builds.cask.co/browse/CDAP-UDUT736

### CODE REVIEW

**Review by ajainarayanan — COMMENTED** [review]
Overall code LGTM. Noticed some things in UI when I pulled the branch,

1. **A brief white background when closing the preview**
preview

2. We don't seem to poll the preview status. When I start the preview and open logs, the logs stays empty even after the preview is complete.

3. (nit) The Get Latest Logs button almost looks invisible. Can we use a different color for selected state?
Screen Shot 2020-06-26 at 10 18 28 AM

File: cdap-ui/app/directives/log-viewer-preview/log-viewer-preview.less [hunk 1]
```diff
@@ -18,24 +18,24 @@
   .canvas-wrapper {
     .top-panel {
       .pipeline-settings {
-        &.pipeline-logs-section {
-          left: 0;
-          max-width: initial;
-          max-height: initial;
+        // &.pipeline-logs-section {
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/log-viewer-preview/log-viewer-preview.less]: nit: Remove?

**Review by ajainarayanan — APPROVED** [review]
👍 LGTM. Just one minor nit.

File: cdap-ui/app/cdap/components/PreviewLogs/index.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+ */
+
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import withStyles, { WithStyles, StyleRules } from '@material-ui/core/styles/withStyles';
+import PreviewDataFetcher from 'components/LogViewer/DataFetcher/PreviewDataFetcher';
+import LogViewer from 'components/LogViewer';
+import ThemeWrapper from 'components/ThemeWrapper';
+
+const height = 'calc(100vh - 48px - 41px - 54px)';
```
Comment (yukiej) [inline cdap-ui/app/cdap/components/PreviewLogs/index.tsx:25]: Maybe add comment about where some of these values are coming from?

**Pull request merged**